### ISSUE TITLE
Build issues -- cannot import urllib

### ISSUE THREAD

**Issue by wasade** [daler/pybedtools#153]
Please see details below, this is on a fresh environment. I also attempted removing six via pip and installing via conda. 

```bash
(py35-base)14:36:19 (omiited@omitted):~$ cat /etc/redhat-release
Red Hat Enterprise Linux Server release 7.1 (Maipo)
(py35-base)14:36:32 (omiited@omitted):~$ conda create --name "test" python=2
Fetching package metadata: ....
Solving package specifications: ..............
Package plan for installation in environment /omitted/miniconda3/envs/test:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    python-2.7.11              |                0        12.0 MB

The following NEW packages will be INSTALLED:

    openssl:    1.0.2d-0
    pip:        7.1.2-py27_0
    python:     2.7.11-0
    readline:   6.2-2
    setuptools: 18.5-py27_0
    sqlite:     3.8.4.1-1
    tk:         8.5.18-0
    wheel:      0.26.0-py27_1
    zlib:       1.2.8-0

Proceed ([y]/n)? y

Fetching packages ...
python-2.7.11- 100% |####################################################| Time: 0:00:10   1.19 MB/s
Extracting packages ...
[      COMPLETE      ]|#######################################################################| 100%
Linking packages ...
[      COMPLETE      ]|#######################################################################| 100%
#
# To activate this environment, use:
# $ source activate test
#
# To deactivate this environment, use:
# $ source deactivate
#
(py35-base)14:37:14 (omiited@omitted):~$ act test
discarding /users/omiited/miniconda3/envs/py35-base/bin from PATH
prepending /users/omiited/miniconda3/envs/test/bin to PATH
(test)14:39:51 (omiited@omitted):~$ pip install pybedtools
Collecting pybedtools
  Downloading pybedtools-0.7.4.tar.gz (12.6MB)
    100% |████████████████████████████████| 12.6MB 25kB/s
Collecting six (from pybedtools)
  Downloading six-1.10.0-py2.py3-none-any.whl
Collecting pysam (from pybedtools)
  Downloading pysam-0.8.4.tar.gz (1.7MB)
    100% |████████████████████████████████| 1.7MB 176kB/s
Building wheels for collected packages: pybedtools, pysam
  Running setup.py bdist_wheel for pybedtools
  Stored in directory: /users/omiited/.cache/pip/wheels/43/15/87/0692a42ebc04a9f818a96aaef6e5a4ad0c7ff2c3c932e8a20f
  Running setup.py bdist_wheel for pysam
  Stored in directory: /users/omiited/.cache/pip/wheels/dc/fe/3c/02893b46df0dda484b02e76a26b8497c8d50450744f2e01101
Successfully built pybedtools pysam
Installing collected packages: six, pysam, pybedtools
Successfully installed pybedtools-0.7.4 pysam-0.8.4 six-1.10.0
(test)14:46:05 (omiited@omitted):~$ pybedtools
Traceback (most recent call last):
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 46, in <module>
    main()
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 24, in main
    mods = [import_module("pybedtools.scripts.%s" % s) for s in scripts]
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 12, in import_module
    __import__("%s" % (name,), globals(), locals(), [], -1)
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/scripts/peak_pie.py", line 14, in <module>
    from six import urllib
ImportError: cannot import name urllib
(test)14:46:09 (omiited@omitted):~$ pip uninstall six
Uninstalling six-1.10.0:
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/DESCRIPTION.rst
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/METADATA
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/RECORD
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/WHEEL
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/metadata.json
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six-1.10.0.dist-info/top_level.txt
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six.py
  /users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/six.pyc
Proceed (y/n)? y
  Successfully uninstalled six-1.10.0
(test)14:46:18 (omiited@omitted):~$ conda install six
Fetching package metadata: ....
Solving package specifications: ................
Package plan for installation in environment /users/omiited/miniconda3/envs/test:

The following NEW packages will be INSTALLED:

    six: 1.10.0-py27_0

Proceed ([y]/n)? y

Linking packages ...
[      COMPLETE      ]|#######################################################################| 100
(test)14:46:24 (omiited@omitted):~$ pybedtools
Traceback (most recent call last):
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 46, in <module>
    main()
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 24, in main
    mods = [import_module("pybedtools.scripts.%s" % s) for s in scripts]
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/__main__.py", line 12, in import_module
    __import__("%s" % (name,), globals(), locals(), [], -1)
  File "/users/omiited/miniconda3/envs/test/lib/python2.7/site-packages/pybedtools/scripts/peak_pie.py", line 14, in <module>
    from six import urllib
ImportError: cannot import name urllib
```

**Comment by wasade** [comment]
cc @coryfunk

**Comment by daler** [comment]
Thanks for reporting this. I had actually forgotten this interface existed for accessing the scripts from the shell. It's not documented and not tested, so I'll consider removing it.

The problem is in the `peak_pie` demo script, which apparently is untested since this import error slipped through the cracks.

I fixed the import error as of a8141da; I will add some tests for `peak_pie` and consider removing the ability to call the `pybedtools` entrypoint from the shell. Actually, I may consider removing the scripts altogether since I'm not aware of anyone using them and they have not been well tested.

**Comment by wasade** [comment]
Great, thank you!
